introductory/src/App.jsx

```diff
@@ -1,8 +1,24 @@
-
+import Button from "./buttonComponent"
 
 function App() {
+
+
+    function handleClick(id){
```

Comment:
Try to be **consistency** in choosing which **type of function** to use between **regular** and **arrow** function